apps/studio/src-commercial/entrypoints/preload.ts

```diff
@@ -39,6 +39,13 @@ export const api = {
   isReady() {
     ipcRenderer.send('ready');
   },
+  rawInvoke(channel: string, ...args: any[]) {
```

Comment:
I don't want to expose raw invoke to the renderer, this will have to go I'm afraid.